anvil/config.py

```diff
@@ -158,10 +173,10 @@ def parse_sample_interval(self, interval: (int, type(None))):
         log.warning(f"Using user specified sample_interval: {interval}")
         return interval
 
-    def parse_bootstrap_n_samples(self, n_samples: int):
+    def parse_n_boot(self, n_boot: int):
```

Comment:
you rename this here but then I see that `observables.two_point_function` is still taking the old input probably needs to be consistent everywhere

Reply:
Ah yes.
We currently have `n_boot` in the example runcard, though maybe that was by accident.
I'll change everything to `n_boot` unless you'd rather keep `bootstrap_n_sample` - it's probably clear enough what it means.

Reply:
not bothered - change to n_boot